### PR TITLE
Update boilerplate template and file headers to use "The Testgrid Authors"

### PR DESCRIPTION
**Pull request by MushuEE**
Update all current headers and boilerplate templates to replace Kubernetes->Testgrid

#137 
/cc @itsazhuhere

### CODE REVIEW

**Comment by MushuEE** [comment]
/hold
@michelle192837 Some of these files were ported over from K8s project. @itsazhuhere mentioned that we could check for Kubetnetes || Testgrid so that Copywrite check would pass for both types. Should that be the route I take? Or move all to Testgrid like this PR is currently?

**Comment by michelle192837** [comment]
We should check for both. We should not be changing the headers on existing files ("Kubernetes Authors" has to remain for files we forked from kubernetes/test-infra; it's only new files that are created from scratch in this repo that we can claim our own copyright on). So this repo reasonably can/should have a mix of different attributions. ^^

File: hack/verify_boilerplate.py [hunk 1]
```diff
@@ -121,7 +123,14 @@ def file_passes(filename, refs, regexs):  # pylint: disable=too-many-locals
     # Replace all occurrences of the regex "2017|2016|2015|2014" with "YEAR"
     when = regexs["date"]
     for idx, datum in enumerate(data):
-        (data[idx], found) = when.subn('YEAR', datum)
+        (data[idx], found) = when.subn("YEAR", datum)
+        if found != 0:
+            break
+
+    # Replace all occurrences of the regex "Testgrid|Kubernetes" with "AUHTOR"
```
Comment (itsazhuhere) [inline hack/verify_boilerplate.py:130]: nit: this should be "AUTHOR"

File: hack/verify_boilerplate.py [hunk 2]
```diff
@@ -24,6 +24,8 @@
 import re
 import sys
 
+AUTHORS = r"Testgrid|Kubernetes"
```
Comment (itsazhuhere) [inline hack/verify_boilerplate.py:27]: I'm pretty sure we want it properly capitalized as "TestGrid"

File: hack/boilerplate/boilerplate.Dockerfile.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright YEAR The Kubernetes Authors.
+# Copyright YEAR The Testgrid Authors.
```
[the remark this answers is not in the document]
Comment (itsazhuhere) [reply hack/boilerplate/boilerplate.Dockerfile.txt]: Oh woops I guess I didn't hit submit for this comment earlier ^.

**Comment by michelle192837** [comment]
Huh, what's with all the proto changes?

**Comment by michelle192837** [comment]
Oh wait, duh, comment changes.

Not sure why types.proto is also getting whitespace changes, though, but not a big deal.

**Pull request merged**